core/src/main/resources/application.conf

```diff
@@ -52,8 +52,14 @@ system {
   // Max number of retries per job that the engine will attempt in case of a retryable failure received from the backend
   max-retries = 10
 
-  //If 'true' then when Cromwell starts up, it tries to restart incomplete workflows
+  // If 'true' then when Cromwell starts up, it tries to restart incomplete workflows
   workflow-restart = true
+
+  // Cromwell will launch up to N submitted workflows at a time. That number is tunable here
```

Comment:
This is a little confusing. I think you mean `will allow up to N submitted workflows to be running concurrently`?